### DISCUSSION TITLE
Detach button from relay Athom 1 gang

### DISCUSSION THREAD

**Discussion by Jodaille** [arendst/Tasmota#15269] [Q&A] [answered]
Hello,

I would like to control an Athom button (wired to a light), with another one.

I would like to detach relay/button from the second one, I don't need relay in this case

I have tried https://tasmota.github.io/docs/Buttons-and-Switches/#detach-switches-with
SetOption114 1
SwitchMode 15

but when I touch the button, the relay activate or desactivate itself.

I am not sure to have well understood the doc, should it be the right command to play ?

Is it due to button/switch difference ? Or maybe it is the Athom wiring that is different ?

**Comment by barbudor** [comment, marked as answer]
You write that you want to dettach a Button but you are using commands that are relatives to Switches

If the template is configured with Button_1, you should use SetOption 73

**Reply by Jodaille** [reply]
Hello,
I am not sure if it is a switch or a button, the template is:
Athom 1Gang switch (0)

I will look at this template.

SetOption 73 seems not to work, the result is:
{"Command":"Error"}

thank you for your answer ! :)

**Reply by barbudor** [reply]
If you purchased an Athom product with Tasmota pre-flashed, please refer to Athom support

Or read again the doc for the correct syntax https://tasmota.github.io/docs/Buttons-and-Switches/#button

**Reply by Jodaille** [reply]
You are right, I have entered:
SetOption 73 1

instead of:
SetOption73 1

I'm a bit ashamed.
now it works, thank you ! :)

**Comment by Jodaille** [comment]
For reference, 192.168.1.235 is IP address of a Sonoff Basic, pressing the button, the relay toggle, Hourra! \o/

`Rule1 ON Button1#Action DO WebSend [192.168.1.235] POWER1 TOGGLE ENDON`
